### ISSUE TITLE
зависимость полей от доставки и оплаты

### ISSUE THREAD

**Issue by Nip4Fun** [alorian/bxorder#2]
Отличный пример компонента!
Хотелось бы еще увидить пример, как получить список доставок зависимых от оплаты, и наоборот, а так же зависимость полей от выбранной доставки.

**Comment by Ruslan-Aleev** [comment]
Да, если есть "зависимости", то коммонент не понимает эти свойства и сбрасывает их значение.
